capsicum-test.cc

```diff
@@ -72,11 +72,14 @@ char ProcessState(int pid) {
   }
   unsigned int count = 0;
   struct procstat *prstat = procstat_open_sysctl();
-  EXPECT_NE(NULL, prstat) << "procstat_open_sysctl failed.";
+  EXPECT_NE(nullptr, prstat) << "procstat_open_sysctl failed.";
   errno = 0;
   struct kinfo_proc *p = procstat_getprocs(prstat, KERN_PROC_PID, pid, &count);
   if (p == NULL || count == 0) {
```

Comment:
How come this one isn't an issue?

Reply:
I think GCC lets you compare to zero/__null, but template instantiations use unsigned long?

Reply:
Or more likely: literal zero is fine, but by the time the template is instatiated this is a pointer-integer comparison.

Reply:
Ah right, testing::internal::CmpHelperNE. Blegh